local-cli/default.config.js

```diff
@@ -34,6 +34,10 @@ var config = {
 };
 
 function getRoots() {
+  var root = process.env.react_native_app_root;
```

Comment:
What's the convention for env variables used elsewhere? Should it be `REACT_NATIVE_APP_ROOT`? Not sure, just checking.